app/views/simple_discussion/forum_threads/_forum_thread.html.erb

```diff
@@ -1,37 +1,42 @@
-<%= cache forum_thread do %>
   <div class="forum-thread">
     <div class="row">
+      <span class="votable-container text-center">
+        <%= render partial: 'shared/voting', locals: { user: current_user, votable: forum_thread } %>
+      </span>
 
-      <div class="col-sm-1 text-center">
+      <%= cache forum_thread do %>
```

Comment:
Moved the cache block opening here because caching the voting button causes that vote count return to the cached value causing inconsistencies.

Reply:
You can move the cache block back up to the top as long as you add `touch: true` to the ForumVote so it updates the ForumThread each time a vote is created or deleted. 👍